packages/renderer/src/lib/ui/StatusDot.svelte

```diff
@@ -0,0 +1,35 @@
+<!-- StatusDot.svelte -->
+<script lang="ts">
+import type { PodInfoContainerUI } from '../pod/PodInfoUI';
+import Tooltip from './Tooltip.svelte';
+
+export let container: PodInfoContainerUI;
+
+// All the possible statuses that will appear for both Pods and Kubernetes
+const colors: Record<string, string> = {
+  // Podman & Kubernetes
+  running: 'bg-green-500', // Green for running
```

Comment:
The only existing colours we have are in StatusIcon, so we should start be matching those, e.g. -400 here (or arguing that we should change both). It doesn't have all these states though, so picking reasonable ones for things that aren't covered should be ok.